src/main/java/org/codehaus/mojo/versions/DisplayDependencyUpdatesMojo.java

```diff
@@ -148,6 +149,15 @@
     @Parameter( property = "verbose", defaultValue = "false" )
     private boolean verbose;
 
+
+    /**
+     * Limit finding available updates to specific artifacts. Defaults to no limit.
```

Comment:
sounds good but can document what could be the content of this. Maybe with samples. Thanks